rosetta-source/src/main/rosetta/event-common-enum.rosetta

```diff
@@ -143,3 +143,8 @@ enum CreditEventTypeEnum: <"Represents the enumerated values to specify a credit
 	RepudiationMoratorium <"The reference entity, or a governmental authority, either refuses to recognise or challenges the validity of one or more obligations of the reference entity, or imposes a moratorium thereby postponing payments on one or more of the obligations of the reference entity. Subject to the default requirement amount. ISDA 2003 Term: Repudiation/Moratorium.">
 	Restructuring <"A restructuring is an event that materially impacts the reference entity's obligations, such as an interest rate reduction, principal reduction, deferral of interest or principal, change in priority ranking, or change in currency or composition of payment. ISDA 2003 Term: Restructuring.">
 	Writedown <"Results from the fact that the underlier writes down its outstanding principal amount.">
+
+enum ValuationMethodEnum: <"Source and method used for the valuation of the transaction by the reporting counterparty.">
```

Comment:
Change the term 'reporting counterparty' in the description to 'valuation party' (we don't want to pin this to a reporting context)